app/models/community_resource.rb

```diff
@@ -31,20 +33,46 @@ def self.published
     )
   end
 
+  def title; description; end
+  def self.matchable; published; end
+
   def published?
     now = Time.current
     is_approved &&
       (publish_from.present? ? publish_from <= now : true) &&
       (publish_until.nil? || now < publish_until)
   end
 
+  def categories_for_tags
+    Category.where(name: tag_list)
```

Comment:
Do we want to add validations? If you publish a community resource and it's missing either any service area or any category tag, it won't show up in the contributions page. So should we validate that if you're publishing a community resource it must have at least one service area and at least one category tag? Doing so makes a lot of sense to me. 

(The only alternative I can think of is adding a special case in the BrowseFilter logic — nor its partner in the Vue side — where it knows that something like "all categories are checked" means that category-less items can be returned. This alternative seems terrible to me as it's both a leaky abstraction and also I can think of a couple ways that it could go surprisingly wrong.)

Reply:
@svileshina mentioned that it's hard to assume we know what's right without talking to an organization about their specifics — and suggested perhaps a warning is better than an outright validation failure

Reply:
I feel like http://c2.com/ppr/checks.html might be relevant here

Reply:
Added validations with some comments in the commit